src/style.css

```
@@ -0,0 +1,89 @@
+
+@font-face {
+  font-family: "Avenir", sans-serif;
+  src: url(../src/fonts/Avenir-Book.ttf);
+}
+
+body {
+  margin: 0;
+  margin-left: 48px;
+  margin-top: 20px;
+  font-weight: 300;
+  font-size: 14px;
+  font-family: Roboto, sans-serif;
+
+}
+
+.input__search {
+  background: url(../src/images/Search.svg) no-repeat;
+  background-position-y: center;
+  background-position-x: 26px;
+  width: 900px;
+}
+
+.default {
+  margin-bottom: 25px;
+}
+
+.search--big {
+  height: 70px;
+  border-radius: 4px;
+  border: 1px solid rgba(225, 231, 237, 1);
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+  padding-left: 62px;
+  background-size: 19px;
+  width: 900px;
```

```suggestion
  width: 100%;
```
width shouldn't be static